bundle.go

```diff
@@ -12,11 +12,6 @@ import (
 // structure. Version bundles are aggregated into a merged structure represented
 // by the Aggregation structure. Also see the Aggregate function.
 type Bundle struct {
-	// Changelogs describe what changes are introduced by the version bundle. Each
-	// version bundle must have at least one changelog entry.
-	//
-	// NOTE that once this property is set it must never change again.
-	Changelogs []Changelog `json:"changelogs" yaml:"changelogs"`
```

Comment:
We do not use these changelogs here anymore but people are frequently confused by their existence. I remove all changelogs from versionbundles here.

I checked `cluster-service` and `api` briefly and could not find any usage of this property which would cause us any trouble.

Reply:
have you also checked happa?

Reply:
It should not use the same library - I need to check if `gsctl` does. My expectation would be that the response from `api` is not really affected.